lib/Constants.js

```diff
@@ -274,3 +273,45 @@ module.exports.Intents = {
     directMessageReactions: 1 << 13,
     directMessageTyping:    1 << 14
 };
+
+module.exports.InteractionFlags = {
+    EPHEMERAL: 1 << 6
+};
+
+module.exports.InteractionTypes = {
+    PING:              1,
+    APPLICATION_COMMAND:     2,
+    MESSAGE_COMPONENT: 3
+};
+
+module.exports.InteractionResponseTypes = {
+    PONG:                                 1,
+    CHANNEL_MESSAGE_WITH_SOURCE:          4,
+    DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE: 5,
+    DEFERRED_UPDATE_MESSAGE:              6,
+    UPDATE_MESSAGE:                       7
+};
+
+module.exports.CommandOptionTypes = {
+    SUB_COMMAND:       1,
+    SUB_COMMAND_GROUP: 2,
+    STRING:            3,
+    INTEGER:           4,
+    BOOLEAN:           5,
+    USER:              6,
+    CHANNEL:           7,
+    ROLE:              8,
+    MENTIONABLE:       9,
+    NUMBER:            10
+};
+
+module.exports.CommandPermissionTypes = {
+    ROLE: 1,
+    USER: 2
+};
+
+module.exports.CommandTypes = {
+    COMMAND: 1,
```

Comment:
This should probally be changed to `CHAT_INPUT` in order to match the documentation

Reply:
Blame Discord